pkg/coscheduling/core/core.go

```diff
@@ -191,16 +191,16 @@ func (pgMgr *PodGroupManager) PostBind(ctx context.Context, pod *corev1.Pod, nod
 	if pgCopy.Status.Phase != pg.Status.Phase {
 		pg, err := pgMgr.pgLister.PodGroups(pgCopy.Namespace).Get(pgCopy.Name)
 		if err != nil {
-			klog.Error(err)
+			klog.ErrorS(err, "pod group", klog.KObj(pgCopy))
```

Comment:
The [guide](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#name-arguments) mentioned to use carmelCase name as the key:

```
Always use lowerCamelCase, for example use containerName and not container name or container_name.
```

So please use `podGroup` as the key (applies elsewhere).